src/main/java/org/spongepowered/common/mixin/core/data/MixinDataHolder.java

```diff
@@ -413,9 +413,4 @@ public DataContainer toContainer() {
         return Collections.emptyList();
     }
 
-    @Override
-    public DataHolder copy() {
```

Comment:
Mixin 0.7 claims that it clashes with lower-level implementations. The override has been migrated to classes further down in the hierarchy where necessary `MixinSpongeUser` and `MixinTileEntity`.